etl/steps/data/garden/papers/2023-11-21/snakebite.excluded_countries.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Why not keeping some of these regions?

Reply:
The IHME regions are kind of confusing, mostly because they are non-overlapping, so Japan and South Korea are in 'High-income Asia Pacific', but not in 'Southeast Asia', which I think is a bit surprising, so I prefer to exclude them.